README.rst

```diff
@@ -3,6 +3,12 @@ frontend-template-application
 
 |license-badge| |status-badge| |ci-badge| |codecov-badge|
 
+⚠️ Warning ⚠️
```

Comment:
I wanted to use [the GH alerts](https://github.com/orgs/community/discussions/16925) for this but I couldn't figure out how to do that in rst.

Reply:
Most of the feapp's have markdown READMEs, it wouldn't be insane to convert this to md imho...